### PR TITLE
schema: acquired for console_session turned into datetime

### LINKED ISSUE #7159: Improve the console session acquired field
As pointed on the PR https://github.com/apache/cloudstack/pull/7132#discussion_r1090566123 it would be good to set the acquired field as a date instead of a boolean

##### ISSUE TYPE
 * Enhancement Request

##### COMPONENT NAME
~~~
Console proxy
~~~

##### CLOUDSTACK VERSION
~~~
4.18
~~~

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Fixes #7159

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: engine/schema/src/main/resources/META-INF/db/schema-41810to41900.sql [hunk 1]
```diff
@@ -19,3 +19,7 @@
 -- Schema upgrade from 4.18.1.0 to 4.19.0.0
 --;
 
+-- Invalidate existing console_session records
+UPDATE `cloud`.`console_session` SET removed=now();
+-- Modify acquired column in console_session to datetime type
+ALTER TABLE `cloud`.`console_session` MODIFY `acquired` datetime COMMENT 'When the session was acquired;
```
Comment (weizhouapache) [inline engine/schema/src/main/resources/META-INF/db/schema-41810to41900.sql:25]: - missing ' as last char

- error when execute the sql statement
```
mysql> ALTER TABLE `cloud`.`console_session` MODIFY `acquired` datetime COMMENT 'When the session was acquired';
ERROR 1292 (22007): Incorrect datetime value: '0' for column 'acquired' at row 1
```

We probably need to remove all records from the table. 
```
mysql> delete from console_session;
Query OK, 51 rows affected (0.01 sec)


mysql> ALTER TABLE `cloud`.`console_session` MODIFY `acquired` datetime COMMENT 'When the session was acquired';
Query OK, 0 rows affected (0.07 sec)
Records: 0  Duplicates: 0  Warnings: 0

```

**Comment by shwstppr** [comment]
@blueorangutan package

**Review by weizhouapache — APPROVED** [review]
Code lgtm

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by rohityadavcloud** [comment]
@blueorangutan test

**Review by nvazquez — APPROVED** [review]

**Pull request merged**